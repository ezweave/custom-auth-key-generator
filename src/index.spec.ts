import { buildKey } from '.';

describe('buildKey', () => {
    it('builds a key', () => {
        expect(
            buildKey({
                path: 'foo/bar',
            })
        ).toMatchSnapshot();
    });
});
