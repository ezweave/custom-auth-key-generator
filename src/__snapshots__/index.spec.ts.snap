// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buildKey builds a key 1`] = `"Somekind of Auth PUBLIC_KEY:AY/lhu5MpnkWksvFfoPtMLq/RkD3aBHGY0ZDL3OOWMAhHBgPkUEf1covCSrPIrYR7UE/Pys+V0EGpBxhS7/Odg=="`;
